src/app/pages/todos/components/todo-card/todo-card.component.html

```diff
@@ -1,14 +1,93 @@
-  <span class="label">{{ buildTranslationKey('status') | translate }}:</span>
-  <span class="info todo-status {{ todo.status }}">{{ buildTranslationKey(todo.status) | translate }}</span>
+<form [formGroup]="form" class="form">
+  <span class="label">{{ buildTranslationKey('status') | translate }}</span>
+  <span class="info todo-status {{ getClass(todo.status) }}">{{ buildTranslationKey(todo.status) | translate }}</span>
```

Comment:
you don't need to pass it inside the template, because you have an access to it in TS.
Create a getter like statusClass